python/ray/tests/test_streaming_generator.py

```diff
@@ -144,96 +144,6 @@ def test_streaming_object_ref_generator_task_failed_unit(mocked_worker):
                 ref = generator._next_sync(timeout_s=0)
 
 
-@pytest.mark.asyncio
-async def test_streaming_object_ref_generator_unit_async(mocked_worker):
-    """
-    Verify the basic case:
-    create a generator -> read values -> nothing more to read -> delete.
-    """
-    c = mocked_worker.core_worker
-    generator_ref = ray.ObjectRef.from_random()
-    generator = StreamingObjectRefGenerator(generator_ref, mocked_worker)
-    c.try_read_next_object_ref_stream.return_value = ray.ObjectRef.nil()
-
-    # Test when there's no new ref, it returns a nil.
-    next_ref = ray.ObjectRef.from_random()
-
-    async def coro_ref():
-        await asyncio.sleep(1)
-        return next_ref
-
-    c.peek_object_ref_stream.return_value = coro_ref()
-    ref = await generator._next_async(timeout_s=0)
-    assert ref.is_nil()
-
-    # When the new ref is available, next should return it.
-    for _ in range(3):
-        next_ref = ray.ObjectRef.from_random()
-
-        async def coro_ref():
-            return next_ref
-
-        c.peek_object_ref_stream.return_value = coro_ref()
-        c.try_read_next_object_ref_stream.return_value = next_ref
-        ref = await generator._next_async(timeout_s=0)
-        assert next_ref == ref
-
-    # When try_read_next_object_ref_stream raises a
-    # ObjectRefStreamEndOfStreamError, it should raise a stop iteration.
-
-    async def coro_ref():
-        return next_ref
-
-    c.peek_object_ref_stream.return_value = coro_ref()
-    generator._generator_ref = coro_ref()
-    c.try_read_next_object_ref_stream.side_effect = ObjectRefStreamEndOfStreamError(
-        ""
-    )  # noqa
-    with pytest.raises(StopAsyncIteration):
-        ref = await generator._next_async(timeout_s=0)
-
-
-@pytest.mark.asyncio
```

Comment:
are you planning to delete this, or is it temporary?

Reply:
These tests failed afterwards. I took a look and I don't think we really need them (they're testing internal behavior).